ghost/admin/app/components/dashboard/charts/anchor-attribution.js

```diff
@@ -811,4 +814,22 @@ export default class Anchor extends Component {
 
         return Math.round((to - from) / from * 100);
     }
+
+    getYAxisMin() {
+        if (this.selectedChartDisplay !== 'mrr') {
+            return 0;
+        }
+        const data = this.chartData.datasets[0].data;
```

Comment:
We should add null or undefined checks for this.chartData and this.chartData.datasets[0] to prevent potential runtime errors when the data is unavailable. Something like this:-

```
if (!this.chartData || !this.chartData.datasets.length) {
        return null;
    }
```